libvirttest/libvirt.go

```diff
@@ -399,6 +400,80 @@ var testRebootReply = []byte{
 	0x00, 0x00, 0x00, 0x00, // status
 }
 
+var testSetBlockIoTuneReply = []byte{
+	0x00, 0x00, 0x00, 0x1c, // length
+	0x20, 0x00, 0x80, 0x86, // program
+	0x00, 0x00, 0x00, 0x01, // version
+	0x00, 0x00, 0x00, 0xfc, // procedure
+	0x00, 0x00, 0x00, 0x00, // type
+	0x00, 0x00, 0x00, 0x00, // serial
+	0x00, 0x00, 0x00, 0x00, // status
+}
+
+// This result block was obtained by calling `fmt.Printf("%#v", r.Payload)` on
+// the result returned by an actual call to GetBlockIoTune, and then adding the
+// standard header to the beginning. The length parameter has to be correct!
+var testGetBlockIoTuneReply = []byte{
+	0x00, 0x00, 0x02, 0xe0, // length
+	0x20, 0x00, 0x80, 0x86, // program
+	0x00, 0x00, 0x00, 0x01, // version
+	0x00, 0x00, 0x00, 0xfd, // procedure
+	0x00, 0x00, 0x00, 0x00, // type
+	0x00, 0x00, 0x00, 0x00, // serial
+	0x00, 0x00, 0x00, 0x00, // status
+	0x0, 0x0, 0x0, 0x13, 0x0, 0x0, 0x0, 0xf,
```

Comment:
can we roughly comment on the data contained below?

Reply:
I can, yes. In general though, it's more valuable to maintainers to tell them how to get these bytes than how to change them - we don't really want devs to construct these test blocks themselves, as that invalidates the test. I'm guessing that tcpdump was used to capture the first few, but some of them have the wrong values for procedure, so you can tell they weren't captured.